config/optional/core.entity_form_display.user.user.default.yml

```diff
@@ -0,0 +1,53 @@
+langcode: en
```

Comment:
Why do we need this file now?

Reply:
Because of the password fields, but I will try to install without providing this explicitely